worker-api/src/main/java/com/hpe/caf/api/worker/WorkerResponse.java

```diff
@@ -40,6 +41,22 @@
      * @param context the new context to add to the task message, can be null
      */
     public WorkerResponse(final String queue, final TaskStatus status, final byte[] data, final String msgType, final int version, final byte[] context)
+    {
+        this(queue, status, data, msgType, version, context, false);
+    }
+
+    /**
+     * Create a new WorkerResponse.
+     *
+     * @param queue the reference to the queue that the response data should be put upon. This can be null if no queue is provided
+     * @param status the status of the message the Worker is returning
+     * @param data the serialised task-specific data returned from the Worker internals
+     * @param msgType the task-specific message classifier
+     * @param version the task-specific message API version
+     * @param context the new context to add to the task message, can be null
+     * @param resetTrackTo indicates if the tracking 'trackTo' field should be reset
+     */
+    public WorkerResponse(final String queue, final TaskStatus status, final byte[] data, final String msgType, final int version, final byte[] context, final boolean resetTrackTo)
```

Comment:
I think this should be more generic.  Rather than having a `resetTrackTo` parameter that sets the `trackTo` pipe to `null` just allow the `trackTo` pipe to be set.  In principle I can think of cases where you'd want to set this to a value other than `null`.